src/rapidata/rapidata_client/feature_flags/feature_flags.py

```diff
@@ -49,16 +55,17 @@ def alert_on_fast_response(self, milliseconds: int):
         self._flags["alert_on_fast_response"] = str(milliseconds)
         return self
 
-    def disable_translation(self, value: bool = True):
-        """Disable automatic translation of all texts in the UI.
+
+    def translation_behaviour(self, behaviour: TranslationBehaviour = TranslationBehaviour.BOTH):
+        """Defines what's the behaviour of the translation in the UI.
```

Comment:
What are the three translating behaviors now? Maybe quickly describe them in the docstring here :)

Reply:
Updated!